src/app/core/components/top-bar/top-bar.component.html

```diff
@@ -0,0 +1,11 @@
+<ng-container *ngIf="currentMode">
+  <alg-left-header *ngIf="fullFrameContent" [compactMode]="true"></alg-left-header>
+  <alg-content-top-bar
+    class="content-top-bar"
+    [fullFrameContent]="fullFrameContent"
+    [currentContent]="currentContent$ | async"
```

Comment:
The input `currentContent` does not look useful, `CurrentContentService` could be directly injected into `ContentTopBarComponent`
It would avoid adding the input `currentContent` in `ContentTopBarComponent` and injecting `CurrentContentService` into `TopBarComponent` since it does not use it.

Reply:
Fixed. I've add delay(0) for fix ExpressionChangedAfterItHasBeenCheckedError